AnnoyancesFilter/sections/popups.txt

```diff
@@ -202,6 +202,7 @@ walletinvestor.com#$#body { overflow: visible!important; }
 academyfx.ru#$#.modal-sale { display: none !important; }
 academyfx.ru#$#body { overflow: visible !important; }
 academyfx.ru#$#html { overflow: visible !important; }
+indianexpress.com##body > div#append_breaking_box
```

Comment:
Not sure if it must be blocked.

Reply:
Same element with id="append_breaking_box" is pretty annoying on mobile layot.